src/AWS.Deploy.Common/CloudApplication.cs

```diff
@@ -27,16 +29,27 @@ public class CloudApplication
         /// </summary>
         public string RecipeId { get; set; }
 
+        /// <summary>
+        /// Last updated time of CloudFormation stack
+        /// </summary>
+        public DateTime? LastUpdatedTime { get; set; }
+
+        /// <summary>
+        /// Indicates whether the Cloud Application has been redeployed from this machine
+        /// </summary>
+        public bool LocallyRedeployed { get; set; }
```

Comment:
This meant to be the current (logged in) user not "machine" in the comment. 

Also, it will make easier to understand if we rename it to `UpdatedByCurrentUser` IMO.

Reply:
Done